### PR TITLE
aws-s3-mv: add page

### PR DESCRIPTION
**Pull request by shermanhui**
This PR addresses #8406 

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Screen Shot 2022-10-20 at 20 17 10

### CODE REVIEW

File: pages/common/aws-s3-mv.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# aws s3 mv
+
+> Move a local file or S3 object to another location locally or in S3.
```
Comment (navarroaxel) [inline pages/common/aws-s3-mv.md:3]:
```suggestion
> Move local files or S3 objects to another location locally or in S3.
```

File: pages/common/aws-s3-mv.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/mv.html>.
+
+- Move a single file to a specified bucket and key:
+
+`aws s3 mv {{file_name}} s3://{{example_bucket_name}}/{{file_name_2}}.txt`
```
Comment (navarroaxel) [inline pages/common/aws-s3-mv.md:8]:
```suggestion
`aws s3 mv {{path/to/local_file}} s3://{{bucket_name}}/{{path/to/remote_file}}`
```

File: pages/common/aws-s3-mv.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# aws s3 mv
+
+> Move a local file or S3 object to another location locally or in S3.
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/mv.html>.
+
+- Move a single file to a specified bucket and key:
```
Comment (navarroaxel) [inline pages/common/aws-s3-mv.md:6]:
```suggestion
- Move a file from local to a specified bucket and key:
```

File: pages/common/aws-s3-mv.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`aws s3 mv {{file_name}} s3://{{example_bucket_name}}/{{file_name_2}}.txt`
+
+- Move a single S3 object to a specified bucket and key:
```
Comment (navarroaxel) [inline pages/common/aws-s3-mv.md:10]:
```suggestion
- Move a specific S3 object into another bucket:
```

File: pages/common/aws-s3-mv.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`aws s3 mv s3://{{example_bucket_name}}/{{file_name}}.txt s3://{{example_bucket_name}}/{{file_name_2}}.txt`
```
Comment (navarroaxel) [inline pages/common/aws-s3-mv.md:12]:
```suggestion
`aws s3 mv s3://{{bucket_name1}}/{{path/to/file}} s3://{{bucket_name2}}/{{path/to/target}}`
```

File: pages/common/aws-s3-mv.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- Move a single S3 object to a specified bucket and retain its original name:
```
Comment (navarroaxel) [inline pages/common/aws-s3-mv.md:14]:
```suggestion
- Move a specific S3 object into another bucket keeping the original name:
```

File: pages/common/aws-s3-mv.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+`aws s3 mv s3://{{example_bucket_name}}/{{file_name}}.txt s3://{{example_bucket_name}}`
```
Comment (navarroaxel) [inline pages/common/aws-s3-mv.md:16]:
```suggestion
`aws s3 mv s3://{{bucket_name1}}/{{path/to/file}} s3://{{bucket_name2}}`
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**